src/Gui/App.luau

```diff
@@ -0,0 +1,55 @@
+local Hoarcekat = script:FindFirstAncestor("Hoarcekat")
+local ReactRoblox = require(Hoarcekat.Packages.ReactRoblox)
+local React = require(Hoarcekat.Packages.React)
+
+local e = React.createElement
+local MainContext = require(Hoarcekat.Plugin.Gui.Contexts.MainContext)
+local StudioTheme = require(Hoarcekat.Plugin.Gui.Contexts.StudioTheme)
+local VerticalSplitter = require(Hoarcekat.Plugin.Gui.Components.VerticalSplitter)
+local Sidebar = require(Hoarcekat.Plugin.Gui.Components.Sidebar)
+local Preview = require(Hoarcekat.Plugin.Gui.Components.Preview)
+
+type Props = {
+	root: ReactRoblox.RootType,
+	plugin: any,
+	children: any?,
```

Comment:
`React.ReactNode` is what you want (don't even need `any`, `nil` is a valid `ReactNode`)

Reply:
Discussed via Discord, will create a `PluginFacade` type for the `plugin` prop and handle other appearances of `any` in prop types with `ReactNode` instead.